Lombiq.VueJs.Samples/Assets/Scripts/VueComponents/business-card.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="card qr-card" v-if="!loading">
+        <div class="card-body" v-if="!error && currentCard">
+            <h5 class="full-name">{{ currentCard.BusinessCard.FirstName.Text }}, {{ currentCard.BusinessCard.LastName.Text }}</h5>
+            <p v-if="currentCard.BusinessCard.Phone.Text">Phone: {{ currentCard.BusinessCard.Phone.Text }}</p>
+            <p v-if="currentCard.BusinessCard.Email.Text">Email: {{ currentCard.BusinessCard.Email.Text }}</p>
+        </div>
+        <div class="card-body" v-else-if="error">
+            <h5>Something went wrong :(</h5>
```

Comment:
Localize and rephrase:
```suggestion
            <h5>T["An error occurred."]</h5>
```